server/src/service/ActivityService.js

```diff
@@ -19,4 +22,23 @@ export class ActivityService extends BaseService {
         const createActivity = this.activityRepository.create(activity);
         await this.activityRepository.save(createActivity);
     }
+
+    async getActivities(userId, boardId) {
+        const board = await this.boardRepository.findOne(boardId);
+        if (!board) throw new EntityNotFoundError();
+
+        const boardService = BoardService.getInstance();
+        await boardService.checkForbidden(userId, boardId);
+
+        const activities = await this.activityRepository.find({ board: boardId });
+        return activities.map((activity) => {
+            return {
+                ...activity,
+                boardId,
+                createdAt: moment(activity.createdAt)
+                    .tz('Asia/Seoul')
+                    .format('YYYY-MM-DD hh:mm:ss'),
```

Comment:
```suggestion
                    .format('YYYY-MM-DD HH:mm:ss'),
```
소문자 hh는 0-12, 대문자 HH는 0-24까지의 숫자를 표현합니다ㅎㅎ

Reply:
반영하였습니당 !!!